test/net/sourceforge/kolmafia/request/RelayRequestTest.java

```diff
@@ -57,4 +72,259 @@ public void exerciseSomeObjectMethodsForCoverage() {
     assertFalse(rr.retryOnTimeout());
     rr.constructURLString("diary.php?textversion=1");
   }
+
+  @Nested
+  class JsonApi {
+    private RelayRequest makeApiRequest(String bodyString) {
```

Comment:
some more tests are necessary. Ones I noticed:
* multiple properties requested
* multiple functions called
* objectType is case insensitive, so either test that or make it case sensitive (possibly preferable, considering I expect all users will be using TypeScript anyway)

there are likely more